Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/CvarAdapter.cpp

```diff
@@ -21,6 +21,15 @@ namespace AZ::DocumentPropertyEditor
         {
         }
 
+        void SetCVarValue(const ConsoleFunctorBase* functor, const AZStd::string& valueString)
+        {
+            auto consoleInterface = AZ::Interface<AZ::IConsole>::Get();
+            AZStd::string consoleCommand(functor->GetName());
+            consoleCommand += " ";
+            consoleCommand += valueString;
+            consoleInterface->PerformCommand(consoleCommand.c_str());
+        }
```

Comment:
Insteead of having an entire function for concatenating the command with the value string together.
It would be better to just use the PerformCommand overload that accepts the command and arguments in separate parameters.

Also the `.c_str()` call is a pessimization here. The AZ::IConsole API accepts a string_view, so there is no need to return a c-string. The c-string would just be constructed into the string_view which would have to call strlen on it to determine the string length again.

```suggestion
            auto consoleInterface = AZ::Interface<AZ::IConsole>::Get()->PerformCommand(consoleCommand, ConsoleCommandContainer{ valueString });
        }
```

Reply:
sure, fixed now..